mpitests/test_apply.py

```diff
@@ -35,22 +35,19 @@
 )
 
 
-def create_random_init_state(numWires, c_dtype, seed_value=48):
-    """Returns a random initial state of a certain type."""
-    np.random.seed(seed_value)
-
+def create_random_init_state(numWires, c_dtype, seed=None):
+    """Returns a random normalized state of c_dtype with 2**numWires elements."""
+    rng = np.random.default_rng(seed)
     r_dtype = np.float64 if c_dtype == np.complex128 else np.float32
 
     num_elements = 2**numWires
-    init_state = np.random.rand(num_elements).astype(r_dtype) + 1j * np.random.rand(
-        num_elements
-    ).astype(r_dtype)
-    scale_sum = np.sqrt(np.sum(np.abs(init_state) ** 2)).astype(r_dtype)
-    init_state = init_state / scale_sum
-    return init_state
+    init_state = rng.random(num_elements).astype(r_dtype) + 1j * rng.random(num_elements).astype(
+        r_dtype
+    )
+    return init_state / np.linalg.norm(init_state)
 
 
-def apply_operation_gates_qnode_param(tol, dev_mpi, operation, par, Wires):
+def apply_operation_gates_qnode_param(tol, dev_mpi, operation, par, Wires, seed=None):
```

Comment:
I just saw it now, but I don't think it's good to use a data type as a variable name (Wires), as it can be confusing and may lead to unexpected behaviour in the future. I guess we don't need to do anything now if we are in a hurry to merge this PR, but please keep track of this.